staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -732,6 +732,9 @@ func watchHandler(start time.Time,
 	stopCh <-chan struct{},
 ) error {
 	eventCount := 0
+	initialEventsEndBookmarkWarningTickerDuration := 10 * time.Second
```

Comment:
nit: let's remove it - there is no way to configure it, so let's just statically set that when constructing ticker in line 966